### PR TITLE
ref: Render OneLogin idp selector with crispy

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry_auth_saml2/onelogin/templates/sentry_auth_onelogin/select-idp.html [hunk 1]
```diff
@@ -15,37 +15,9 @@
...
-    <input required="required"
-      class="textinput textInput form-control"
-      id="metadata_url"
-      name="metadata_url"
-      type="text"
-      value="{{ metadata_url }}" />
-  </div>
+<form method="post" class="form-stacked">
+  {% include "sentry/partial/form_base.html" %}
+  <input type="hidden" name="provider" value="okta" />
```
Comment (mattrobenolt) [inline src/sentry_auth_saml2/onelogin/templates/sentry_auth_onelogin/select-idp.html]: Shouldn't this be `plugin.slug` ?

Comment (evanpurkhiser) [reply src/sentry_auth_saml2/onelogin/templates/sentry_auth_onelogin/select-idp.html]: oh, the value is wrong too. Yeah it might as well just be that.

Comment (evanpurkhiser) [reply src/sentry_auth_saml2/onelogin/templates/sentry_auth_onelogin/select-idp.html]: Actually sorry no, I don't think so. Unless I'm missing something we don't pass down the 'plugin' when rendering this, and I'm not even sure what that hidden input was doing before.

**Comment by evanpurkhiser** [comment]
whoops, that typo commit was a bit more than what should be in there  :)

**Review by mattrobenolt — APPROVED** [review]

**Pull request merged**